docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+  db:
+    environment:
+      POSTGRES_DB: imagetagger
+      POSTGRES_USER: imagetagger
+      POSTGRES_PASSWORD: imagetagger
+    # restart: always
+    image: postgres
+    expose:
+      - "5432"
+  web:
+    environment:
+      DATABASE_URL: postgres://imagetagger:imagetagger@db/imagetagger
+      DJANGO_MANAGEPY_MIGRATE: "off"
+      DJANGO_MANAGEPY_COLLECTSTATIC: "off"
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    command: python3 /imagetagger/imagetagger/manage.py runserver 0.0.0.0:8080
```

Comment:
Using a virtual environment within the container might prove advantageous as well.